index.html

```diff
@@ -0,0 +1,156 @@
+            <li>
+              <button class="header-btn" type="submit">Get a ticket</button>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="event-main">
+        <section class="event-brif">
+          <h2 class="event-title-main">Egypt: The Time of Pharaohs</h2>
+          <h1 class="event-title-second">
+            Exhibition 2022 &mdash; London: Ancient Egypt
+          </h1>
+          <p class="event-description">
+            London Exhibition 2022 will take visitors into the distant past,
+            exploring the myths and realities of ancient Egyptian culture.
+          </p>
+          <p class="event-date">2022.10.10(Mon) ~ 23(Sun)</p>
+
+          <p class="event-location">
+            @ Saatchi Gallery, Duke of York's HQ, King's Rd, London SW3 4RY
+          </p>
+        </section>
+        <section class="main-program">
+          <h3 class="title">Main Program</h3>
+          <hr class="big-title-bar" />
+          <ul class="main-program-list">
+            <li class="main-program-item">
+              <i class="ph-line-segments main-icon"></i>
+              <h4>Sections</h4>
+              <p>
+                Have a walk thru our different Exhibition sections and
+                experience the Ancient Egypt vibe.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-chalkboard-teacher main-icon"></i>
+              <h4>Lecture</h4>
+              <p>
+                Listen to the speakers, experienced archaeologists revelling in
+                the great mysteries of Ancient Egypt.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-aperture main-icon"></i>
+              <h4>Photography</h4>
+              <p>
+                Famous photographers share their work with photos pyramids,
+                temples, the Nile etc.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-chats main-icon"></i>
+              <h4>Forum</h4>
+              <p>
+                Join the Forum to connect with other scholars passionate about
+                this ancient civilization.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-globe main-icon"></i>
+              <h4>Next</h4>
+              <p>
+                Find out about future exhibitions in London and across the
+                globe.
+              </p>
+            </li>
+          </ul>
+          <button type="submit" class="main-btn">Get a ticket</button>
+          <a class="see-program" href="#">See the whole program</a>
+        </section>
+        <section class="exhib-sections">
+          <h3 class="title">Exhibition Sections</h3>
+          <hr class="big-title-bar" />
+          <ul class="exhib-section-list"></ul>
+        </section>
+      </main>
+      <footer class="footer">
```

Comment:
The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure that:
- [ ]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.